test/tools/yulInterpreter/Interpreter.h

```diff
@@ -96,6 +96,12 @@ struct InterpreterState
 	void dumpTraceAndState(std::ostream& _out) const;
 };
 
+struct Scope
+{
+	std::map<YulString, FunctionDefinition const*> m_functions;
```

Comment:
No `m_` prefix for public struct members.

Reply:
Can you use `std::set<YulString>` here?

Reply:
I need the definition to pass it to the `Interpreter` instance that we create

Reply:
I'm just wondering because this is more or less duplicated in the `m_functions` member of the interpreter.

Reply:
So maybe that member can be removed?